auto_forwarder.py

```
@@ -1354,4 +1718,265 @@ def _refresh_settings_ui(self):
+        """Processes unread messages for all rules that allow batch processing."""
+        def process_in_background():
+            try:
+                count = 0
+                for chat_id, rule in self.forwarding_rules.items():
+                    if rule.get("enabled", False) and not rule.get("batch_ignore", False):
+                        self._process_unread_messages(chat_id, rule)
+                        count += 1
+                        time.sleep(0.5)  # Small delay between chats
+                
+                run_on_ui_thread(lambda: BulletinHelper.show_success(f"Queued unread processing for {count} chats"))
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)